packages/pg/lib/connection.js

```diff
@@ -85,11 +85,13 @@ Connection.prototype.connect = function (port, host) {
   this.stream.once('data', function (buffer) {
     var responseCode = buffer.toString('utf8')
     switch (responseCode) {
-      case 'N': // Server does not support SSL connections
-        return self.emit('error', new Error('The server does not support SSL connections'))
       case 'S': // Server supports SSL connections, continue with a secure connection
         break
+      case 'N': // Server does not support SSL connections
+        self.end()
```

Comment:
It looks like this might make the connection emit both `'error'` and `'end'`. Is that okay? I don’t think it can in any other situations, but I might be wrong.

Reply:
i think that's fine really, but I can change it